puente-analytics-service/scripts/postgres.py

```diff
@@ -0,0 +1,138 @@
+import psycopg2
+import sys
+import boto3
+import os
+import secretz
+
+
+
+#gets the credentials from .aws/credentials
+#session = boto3.Session(profile_name='RDSCreds')
+client = boto3.client('rds', aws_access_key_id=secretz.AWS_ACCESS_KEY_ID,
+    aws_secret_access_key=secretz.AWS_SECRET_ACCESS_KEY, region_name=secretz.REGION)
+
+try:
+    conn = psycopg2.connect(host=secretz.ENDPOINT, port=secretz.PORT, database=secretz.DBNAME, user=secretz.USER, password=secretz.PW, sslrootcert="SSLCERTIFICATE")
+    cur = conn.cursor()
```

Comment:
Generally best practice to wrap database connections and cursors in a `with` clause to ensure the connection gets closed.

sorry for the unsolicited code review, I am bored